walkeeper/src/bin/safekeeper.rs

```diff
@@ -43,6 +43,9 @@ fn main() -> Result<()> {
                 .takes_value(true)
                 .help(formatcp!("http endpoint address for metrics on ip:port (default: {DEFAULT_HTTP_LISTEN_ADDR})")),
         )
+        // FIXME this argument is no longer needed since pageserver address is forwarded from compute.
+        // However because this argument is in use by console's e2e tests lets keep it for now and remove separately.
+        // So currently it is a noop.
```

Comment:
It is easier to remove it from console e2e tests first and then remove from safekeeper. A bit of a circular dependency between repos